### PR TITLE
Binance websocket integration tests

### PR DESCRIPTION
**Pull request by karen-stepanyan**
## Changes

- Added websocket integration tests for Binance EA

## Steps to Test

`yarn test packages/sources/binance/test/integration`

## Quality Assurance

- [ ] Ran `yarn changeset` if adapter source code was changed
- [ ] If a new adapter was made, or an existing one was modified so that its environment variables have changed, update the relevant `<ADAPTER_PACKAGE>/schemas/env.json` and `<ADAPTER_PACKAGE>/README.md`
- [ ] If a new adapter was made, or an existing one was modified so that its environment variables have changed, update the relevant `infra-k8s` configuration file.
- [ ] The branch naming follows git flow (`feature/x`, `chore/x`, `release/x`, `hotfix/x`, `fix/x`) or is created from Clubhouse/Shortcut
- [ ] This is related to a maximum of one Clubhouse/Shortcut story or GitHub issue
- [x] Types are safe (avoid TypeScript/TSLint features like any and disable, instead use more specific types)

### CODE REVIEW

**Review by alejoberardino — DISMISSED** [review]
LGTM :rocket:

File: packages/sources/binance/test/integration/adapter.test.ts [hunk 1]
```diff
@@ -76,3 +92,85 @@ describe('execute', () => {
...
+
+describe('websocket', () => {
+  let mockedWsServer: InstanceType<typeof MockWsServer>
+  let server: http.Server
+  let req: SuperTest<Test>
+
+  let oldEnv: NodeJS.ProcessEnv
+  beforeAll(async () => {
+    if (!process.env.RECORD) {
+      mockedWsServer = mockWebSocketServer('wss://stream.binance.com:9443/ws')
```
Comment (austinborn) [inline packages/sources/binance/test/integration/adapter.test.ts:104]: Suggestion: you can pull the ws endpoint from `DEFAULT_WS_API_ENDPOINT` in `config.ts`

**Review by austinborn — DISMISSED** [review]

File: packages/sources/binance/test/integration/adapter.test.ts [hunk 2]
```diff
@@ -76,3 +92,85 @@ describe('execute', () => {
...
+          .post('/')
+          .send(data)
+          .set('Accept', '*/*')
+          .set('Content-Type', 'application/json')
+          .expect('Content-Type', /json/)
+          .expect(200)
+
+      // We don't care about the first response, coming from http request
+      // This first request will start both batch warmer & websocket
+      // TODO: Validate just in case?
```
Comment (austinborn) [inline packages/sources/binance/test/integration/adapter.test.ts:157]: @alejoberardino Do we need this TODO here?

Comment (karen-stepanyan) [reply packages/sources/binance/test/integration/adapter.test.ts:157]: yeah i think this should be removed

**Review by eddiejhong — APPROVED** [review]

**Pull request merged**